src/data/template/static/js/control/mousepos.js

```diff
@@ -15,6 +15,7 @@ MousePosControl.prototype.create = function(wrapper) {
 			var xzy = ui.latLngToMC(event.latlng, 64);
 			document.getElementById("mouse-move-div").innerHTML = '<div class="btn-group" role="group">'
 			+ '<button type="button" class="btn btn-default">' + "X: " + Math.round(xzy[0]) + '</button>'
+			+ '<button type="button" class="btn btn-default">' + "Y: " + Math.round(xzy[2]) + '</button>'
```

Comment:
I don't really agree with this. I haven't seen it in the reversed order anywhere else, but I have made the changes nonetheless.

Sorry I'm working on this so slowly. I have a lot going on right now.